test/cypress/integration/stripe-setup/responsible_person_spec.js

```diff
@@ -0,0 +1,518 @@
+'use strict'
+
+const commonStubs = require('../../utils/common_stubs')
+
+describe('Stripe setup: responsible person page', () => {
+  const gatewayAccountId = 42
+  const userExternalId = 'userExternalId'
+
+  const firstName = 'William'
+  const typedFirstName = 'William '
+  const lastName = 'Benn'
+  const typedLastName = ' Benn'
+  const addressLine1 = '52 Festive Road'
+  const typedAddressLine1 = ' 52 Festive Road'
+  const addressLine2 = 'Putney'
+  const typedAddressLine2 = 'Putney '
+  const city = 'London'
+  const typedCity = 'London '
+  const postcode = 'SW15 1LP'
+  const typedPostcode = 'sw151lp '
+  const dobDay = '25'
+  const typedDobDay = '25 '
+  const dobMonth = '02'
+  const typedDobMonth = ' 02'
+  const dobYear = '1971'
+  const typedDobYear = '1971 '
+  const friendlyDob = '25 February 1971'
+  const longText = 'This text is 300 ...............................................................................' +
+    '...............................................................................................................' +
+    '............................................................................ characters long'
+
+  const stubStripeAccountGet = function stubStripeAccountGet (stripeAccountId) {
+    return {
+      name: 'getStripeAccountSuccess',
+      opts: {
+        gateway_account_id: gatewayAccountId,
+        stripe_account_id: stripeAccountId
+      }
+    }
+  }
+
+  const stubStripeSetupGet = function stubStripeSetupGet (responsiblePersonCompleted) {
+    return {
+      name: 'getGatewayAccountStripeSetupSuccess',
+      opts: {
+        gateway_account_id: gatewayAccountId,
+        responsible_person: responsiblePersonCompleted
+      }
+    }
+  }
+
+  const stubStripeSetupGetForMultipleCalls = function stubStripeSetupGetForMultipleCalls (...responsiblePersonCompleted) {
+    const data = responsiblePersonCompleted.map(completed => (
```

Comment:
Nice 👍 , we should use similar style in the other tests as well

Reply:
I’m actually quite proud of this!